commerce-account-admin-web/src/main/resources/META-INF/resources/account/detail.jsp

```diff
@@ -83,6 +83,36 @@ long commerceAccountId = commerceAccountAdminDisplayContext.getCommerceAccountId
 								/>
 							</c:if>
 						</div>
+
+						<c:if test="<%= commerceAccount != null %>">
+							<aui:select label="default-billing" name="defaultBillingAddressId" showEmptyOption="<%= true %>">
+
+								<%
+								for (CommerceAddress billingAddress : commerceAccountAdminDisplayContext.getBillingCommerceAddresses()) {
```

Comment:
_Ideally_, this variable would be named `billingCommerceAddress` rather than `billingAddress`. Usually this isn't a huge deal, but the fact that there's a portal entity `Address` makes this a potential source of confusion in the future, so I think it's worth making the change here.